xcodecache/swiftpm_cache_test.go

```diff
@@ -50,7 +49,7 @@ func TestCollectSwiftPackages(t *testing.T) {
 		t.Skip("Skipping integration test as -short flag is set.")
 	}
 
-	tempDir, err := ioutil.TempDir("", "")
+	tempDir, err := os.MkdirTemp("", "")
```

Comment:
I do not know if it can be used here or not but not so long ago I learned that you can create temporary directories for the tests with `t.TempDir()` and this will be cleaned up once the test is done.

Reply:
Ohh, I didn't know about it, thanks for sharing! 
Updated in every test: https://github.com/bitrise-io/go-xcode/pull/247/commits/743c069bc48c07e36ce899d6889b4467f82bd862